env0/resource_environment.go

```diff
@@ -77,12 +77,14 @@ func resourceEnvironment() *schema.Resource {
 				Type:        schema.TypeBool,
 				Description: "redeploy only on path changes only",
 				Optional:    true,
+				Default:     false,
```

Comment:
it's supposed to be true, but if the user set that to false and no glob pattern, then it should be anyChange no?
@GiliFaroEnv0

Reply:
if we get to that we need to fetch the default from the BE

Reply:
@liranfarage89 since we want the default behavior to be "on any change" this need to be false

Reply:
on any change is not the default behavior as far as I know..

Reply:
check the ui and see

Reply:
@GiliFaroEnv0